### PR TITLE
Update Grafana Labs distribution status

### PR DESCRIPTION
**Pull request by tpaschalis**
Hi! This PR marks Grafana Labs as a vendor offering a distribution on the OpenTelemetry vendor list.

Link to your distribution, if applicable:
https://github.com/grafana/agent

In the past year, we've been working on enabling Grafana Agent to work natively with OTLP, and offering multiple OTel collector components as a distribution ([example](https://github.com/open-telemetry/opentelemetry-collector-contrib/commit/55e2365e56a7359ad580f31bc4fcaeea2e23c27e)).

We're also maintaining two instrumentation distros in:
https://github.com/grafana/grafana-opentelemetry-java
https://github.com/grafana/grafana-opentelemetry-dotnet

Link that proves that your offering is open source, if applicable. An open source distribution does not qualify your offering to be marked “open source”.

The mentioned projects all use the Apache License 2.0
https://github.com/grafana/agent/blob/main/LICENSE
https://github.com/grafana/grafana-opentelemetry-java/blob/main/LICENSE
https://github.com/grafana/grafana-opentelemetry-dotnet/blob/main/LICENSE

GitHub handle or email address as a point of contact so that we can reach out in case we have question
@jpkrohling 
@gouthamve 
@richardqlam
@rfratto
@matt-hensley

### CODE REVIEW

File: data/ecosystem/vendors.yaml [hunk 1]
```diff
@@ -91,7 +91,7 @@
   oss: false
   commercial: true
 - name: Grafana Labs
-  distribution: false
+  distribution: true
   nativeOTLP: true
   url: 'https://grafana.com/oss/opentelemetry/'
   contact: ''
```
Comment (svrnm) [inline data/ecosystem/vendors.yaml:97]: While updating this, can you please also list a contact, thanks!

**Review by svrnm — CHANGES_REQUESTED** [review]

File: data/ecosystem/vendors.yaml (hunk 1 above)
Comment (tpaschalis) [reply data/ecosystem/vendors.yaml:97]: Yeah, adding @jpkrohling's github account here.

Comment (svrnm) [reply data/ecosystem/vendors.yaml]: @jpkrohling: please approve this PR so I know you are OK with that:)

Comment (jpkrohling) [reply data/ecosystem/vendors.yaml]: Done.

**Review by svrnm — APPROVED** [review]

**Comment by svrnm** [comment]
@tpaschalis you created this fork without the permissions of maintainers to push commits to it, that way I am not able to update the branch to be in sync with the main branch. Please take care that the PR is in sync such that we can merge it

**Comment by tpaschalis** [comment]
@srikanthccv apologies, I read that in the contribution guide and missed the step.

I've updated this branch with `main`.

**Comment by svrnm** [comment]
> @svrnm apologies, I read that in the contribution guide but missed the step.
> 
> I've updated this branch with main.

No worries, as a rule of thumb, do not fork a repo within an organization if you want to do an upstream PR, because they default to now providing write permissions

**Pull request merged**